src/chocolatey/infrastructure.app/commands/ChocolateyOutdatedCommand.cs

```diff
@@ -50,7 +50,10 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      option => configuration.SourceCommand.Certificate = option.remove_surrounding_quotes())
                 .Add("cp=|certpassword=",
                      "Certificate Password - the client certificate's password to the source. Defaults to empty. Available in 0.9.10+.",
-                     option => configuration.SourceCommand.CertificatePassword = option.remove_surrounding_quotes())
+                     option => configuration.SourceCommand.CertificatePassword = option.remove_surrounding_quotes())                
+                .Add("ip|ignore-pinned",

```

Comment:
Let's not have a short version of this. At least for now. Just `--ignore-pinned`.